src/plugin.json

```diff
@@ -73,6 +78,224 @@
       "path": "datasource/plugin.json"
     }
   ],
+  "roles": [
+    {
+      "role": {
+        "name": "Checks reader",
+        "description": "Read checks in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+
+          { "action": "grafana-synthetic-monitoring-app.checks:read" }
+        ]
+      },
+      "grants": ["Viewer"]
+    },
+    {
+      "role": {
+        "name": "Checks writer",
+        "description": "Create, edit and delete checks in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+          { "action": "grafana-synthetic-monitoring-app:write" },
```

Comment:
As above I'm not sure what the generic write action gives the user.

Reply:
They are actions I added so that they can be used in `datasource/plugin.json` file https://github.com/grafana/synthetic-monitoring-app/pull/982/files/0c573fe1059ca32027fbc2755ec98e84126d87a1#diff-b692ddd2f38b511f155f18798f8101fa7287f2949c496b715163908296df7effR48

Before, we were using org roles to restrict those operations (actually, just `Editor`), so I had to come up with new custom roles that would be appropriate for those endpoints.